### PR TITLE
Run our unit tests in a GitHub workflow

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

**Comment by martinpitt** [comment]
If you like this, I'll switch the required tests around. Note that OCP4 is currently down, so the webhook is broken, and the old "hosts" test doesn't currently get run.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: test
```
Comment (marusak) [inline .github/workflows/test.yml:1]: `test / tests (pull_request) ` it shows up like this in the list of tests. Could we give it better name? (Not that I know of any :) )

**Review by marusak — DISMISSED** [review]
Nice!
I like the idea that we run as much as we can outside of our own infra and this is a great candidate!
One nitpick, but not really a blocker :)

File: .github/workflows/test.yml (hunk 1 above)
Comment (martinpitt) [reply .github/workflows/test.yml:1]: Indeed - as per IRC, I renamed it to "bots", to coincide with `./test-bots` (the script that runs).

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**